powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -790,7 +790,8 @@ private void approveConsent(final String processId) throws Exception {
                 .findFirst()
                 .orElse("error - no consent found");
 
-        assertThat(consentText, startsWith("<html><body><h1>Lorem ipsum</h1>Lorem ipsum dolor sit amet"));
+        assertThat(consentText, startsWith("<html>"));
+        assertThat(consentText, containsString("Souhlasím, aby Raiffeisenbank a.s. zpracovávala biometrické údaje"));
```

Comment:
This repository is public, please avoid using client names or Czech language in tests.

Reply:
I see. But what to do with this PR? I guess it should not be merged as well, because repo `enrollment-server-wultra` is public too. https://github.com/wultra/enrollment-server-wultra/pull/33

Reply:
Mock consent should be generic.

Reply:
Ok, lets test only `startsWith("<html>"`